alvr/session/src/settings.rs

```diff
@@ -1565,6 +1581,16 @@ pub fn session_settings_default() -> SettingsDefault {
                     tracked: true,
                 },
             },
+            vmc: SwitchDefault {
```

Comment:
The only debate would be whether this is the right tab for those settings to be in. Connections might make more sense

Reply:
I'm going to move this again as soon as we merge this